docs-site/docs/docs/about-menu.md

```diff
@@ -0,0 +1,43 @@
+The about menu provides a deeper view into your instance of Polynote. 
```

Comment:
Can't hurt to write something about how to access this menu, as people might go straight to this page. 

Just a suggestion off the top of my head, please wordsmith to your liking (the link/image are definitely wrong) 😄 

```suggestion
The about menu provides a deeper view into your instance of Polynote. It can be accessed from the [toolbar](toolbar/#about-section): 

![Toolbar About Section](images/about-toolbar-image.whatever)
```

You can just copy the image from https://polynote.org/latest/docs/toolbar/#about-section (and cross-link back to that as well?)